netcontrold/tests/lib/test_util.py

```diff
@@ -262,3 +263,38 @@ def test_rr_cpu_in_numa_positive(self, mock_open):
         out = util.rr_cpu_in_numa()
         expected = [0, 2, 1, 3]
         self.assertEqual(out, expected)
+
+
+class DummyNcdThread(util.Thread):
+    def __init__(self, eobj):
+        util.Thread.__init__(self, eobj)
+
+    def run(self):
+        self.dumyVariable = 1
```

Comment:
Can you correct the spelling for dumyvariable ? .. you could just say self.dummy as it is itself a variable.